conan/tools/microsoft/toolchain.py

```diff
@@ -163,7 +167,8 @@ def format_macro(key, value):
             "runtime_library": runtime_library,
             "toolset": toolset,
             "compile_options": compile_options,
-            "parallel": parallel
+            "parallel": parallel,
+            "properties": self.properties
```

Comment:
```suggestion
            "properties": self.properties,
```
Next time it will only be a one line diff